en/docs/services/create-your-first-service.md

```diff
@@ -153,10 +153,12 @@ To deploy the `covid-stats` service, follow this procedure:
 
     ![Deployment Logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
```

Comment:
Need to follow sentence-style for alt text.

Reply:
Corrected this.